lib/plugins/create/templates/plugin/index.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+// https://github.com/serverless/serverless/blob/master/docs/04-extending-serverless/01-creating-plugins.md
+class ServerlessPlugin {
+  constructor(serverless, options) {
+    this.serverless = serverless;
+    this.options = options;
+
+    this.commands = {
+      deploy: {
+        lifecycleEvents: [
+          'functions',
+        ],
+        options: {
+          function: {
+            usage:
+              'Specify the function you want to deploy '
+              + '(e.g. "--function myFunction" or "-f myFunction")',
+            required: true,
+            shortcut: 'f',
+          },
+        },
+      },
+    };
+
+    this.hooks = {
```

Comment:
Can you add a before hook for the hello and an after hook for the world event, so users can also see they can use before/after hooks